package.json

```diff
@@ -113,11 +113,11 @@
     "@babel/plugin-transform-object-assign": "^7.2.0",
     "@babel/preset-env": "^7.4.4",
     "@babel/preset-react": "^7.0.0",
-    "babel-eslint": "^8.2.1",
+    "babel-eslint": "^10.0.1",
     "babel-jest": "^24.8.0",
     "check-installed-dependencies": "^1.0.0",
     "core-js": "^3.1.3",
-    "danger": "^3.7.14",
+    "danger": "^8.0.0",
```

Comment:
should we remove danger and add the github pr bot since this isn't a mono-repo? @bjankord

Reply:
Yeah, it would be one less dependency we have to worry about. 👍

Reply:
Removed: [c1dabdf](https://github.com/cerner/terra-dev-site/pull/171/commits/c1dabdf8e7c8d0869db48c27572985e3ae9e9cb20)